metrics/factory.go

```diff
@@ -8,6 +8,7 @@ import (
 	"sync"
 
 	"github.com/cashapp/blip"
+	autoincrement "github.com/cashapp/blip/metrics/auto_increment"
```

Comment:
`s/auto.increment/autoinc/g`.  "Autoinc" is standard MySQL vernacular. Plus, standardizing on this avoids this problem (next comment)...